src/Console/Program.cs

```diff
@@ -8,13 +8,15 @@ class Program
         static async Task<int> Main(string[] args)
         {
             var rootCommand = new RootCommand("Client command line application to interact with CustomerPortal DevOps Center");
-            rootCommand.AddCommand(new PublishCommand());
+            rootCommand.AddCommand(new PublishCommand());
+            rootCommand.AddCommand(new DeployAgentCommand());
             rootCommand.AddCommand(new DeployCommand());
             rootCommand.AddCommand(new LoginCommand());
             rootCommand.AddCommand(new CreateInfrastructureFromTemplateCommand());
             rootCommand.AddCommand(new CreateInfrastructureCommand());
             rootCommand.AddCommand(new CheckAgentConnectionCommand());
             rootCommand.AddCommand(new PublishPackageCommand());
+           
```

Comment:
Does the order of these commands has any meaning? If not maybe put it alphabeticaly